openweathermap-ios/openweathermap-iosTests/BookmarkLocation/Manager/BookmarkLocationManagerTest.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  BookmarkLocationServiceTest.swift
+//  openweathermap-ios
+//
+//  Created by Cassio Sousa on 24/04/20.
+//
+
+import XCTest
+
+class BookmarkLocationManagerTest: XCTestCase {
+
+    private var bookmarkLocationManager = BookmarkLocationManager()
```

Comment:
```suggestion
    private let sut = BookmarkLocationManager()
```

`sut` means System Under Test.

Reply:
Fixed in b6bb083